README.md

```diff
@@ -15,6 +15,49 @@ def deps do
 end
 ```
 
+## Usage
+
+If the data is valid, the function returns {:ok, data}. The data returned will be coerced according to the schema defined.
```

Comment:
Can we wrap the {:ok, data} with backticks to make it `{:ok, data}`?

Reply:
Also, let's add a line before this one that gives a general description of the main function that will be used (since we are referring to "the function" here, but it has not been mentioned anywhere). Something along the lines of:

"Litmus validates data against a predefined schema with the `Litmus.validate/2` function".

Reply:
The main function will be listed in the docs. Do we want to repeat it? Take a look at the local docs generated when running `mix docs` to see what it will look like.